src/app/pages/AddTopic/AddTopic.tsx

```diff
@@ -57,3 +99,28 @@ const Header = styled.h1`
   align-self: stretch;
   font-size: 4rem;
 `
+
+const TopicFormContainer = styled.form`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
Do u want the Form content box to have the full width? I can't oben your Heroku deployment, i just screwed up my css with this values once, so i'm asking :)